cmd/eval_test.go

```diff
@@ -29,6 +29,50 @@ import (
 	"github.com/open-policy-agent/opa/util/test"
 )
 
+func TestEvalWithIllegalUnknownArgs(t *testing.T) {
+
+	tests := []struct {
+		name        string
+		unknowns    string
+		expectedErr error
+	}{
+		{
+			name:        "happy path: passing ref as unknown",
+			unknowns:    "data.posts",
+			expectedErr: nil,
+		},
```

Comment:
Please, could you add cases for these?
- `input`
- `input.users`
just to be sure that it'll all end up a ref.

Reply:
I've made the required changes in the test cases. could you plz review